### ISSUE TITLE
Text not showing on Persistent Notifications

### ISSUE THREAD

**Issue by guardianbs** [home-assistant/frontend#17696]
### The problem

Strange problem relating to a specific browser.

Since updating to 2023.7.1 the text in Persistent Notifications is not displayed in the HA Notifications sidebar. Headings are shown but no text.

Text displays fine on Edge and Chrome browsers but does not display on LG TV browsers. I have tried 2 LG TVs with different (uptodate) WebOs releases and they both have the same problem.

All LG TVs displayed Persistent Notifications text on HA releases up to 2023.6.3.

Something must have changed between 2023.6.3 and 2023.7.1 to prevent the display of the Notification text on LG browsers.

### What version of Home Assistant Core has the issue?

2023.7.1 onwards

### What was the last working version of Home Assistant Core?

2023.6.3

### What type of installation are you running?

Home Assistant OS

### Integration causing the issue

Persistent Notification

### Link to integration documentation on our website

https://www.home-assistant.io/integrations/persistent_notification/

### Anything in the logs that might be useful for us?

```txt
Message in home-assistant.log repeated many times

2023-08-23 16:04:10.140 ERROR (MainThread) [frontend.js.latest.202307051] :0:0 Script error.
```

### Additional information

Display should be

![image](https://github.com/home-assistant/core/assets/71522772/b30e3d89-b673-44ca-978a-73963515d1d5)

Display on version 2023.7.1 onwards

![image](https://github.com/home-assistant/core/assets/71522772/21a982c4-7107-447e-98c0-90815ab159c9)

**Comment by guardianbs** [comment]
This problem which only started in HA version 2023.7.1 still persists with the latest version of HA and renders HA unusable on LG web browsers.
Core
2023.11.2
Supervisor
2023.11.3
Operating System
11.1
Frontend
20231030.2

**Comment by guardianbs** [comment]
August, September, October, November, December, January, February - and problem persists on 2024.2.1

HA continues to be unusable with WebOs browser on LG TV.

**Comment by guardianbs** [comment]
HA Core 2024.3.1 and problem persists - when will anybody ever fix this ?

HA continues to be unusable with WebOs browser on LG TV.

**Comment by silamon** [comment]
Please consider to close either this or #17236, since those 2 issues are duplicates of each other. 

Are you able to view the persistent notifications on the [demo page](https://demo.home-assistant.io/)? 
Can you display a [markdown card](https://www.home-assistant.io/dashboards/markdown/) correctly?

**Comment by guardianbs** [comment]
Hi, Thanks for looking at this.

Demo Page: No, the Demo page has the same problem The 'Motion Detected!' title appears and the time 'Now' appears but the text 'There was motion detected in the backyard.' is NOT shown.

![image](https://github.com/home-assistant/frontend/assets/71522772/3534dfcd-c524-486b-b508-7f033ccaf56e)

shows as

![image](https://github.com/home-assistant/frontend/assets/71522772/010cc197-8e5e-4d26-a4e6-8d2c63a27ed9)


Markdown Card: I created a markdown card with 

![image](https://github.com/home-assistant/frontend/assets/71522772/72e81b28-77e6-46e3-87db-30ef7dccea68)

shows as

![image](https://github.com/home-assistant/frontend/assets/71522772/8daeaede-8349-43c7-9a35-a9be58ee5aee)

Both displays work ok on Chrome and Edge on a desktop PC.

The only error in the log was

2024-03-17 19:08:28.252 ERROR (MainThread) [frontend.js.latest.202403070] Uncaught error from Chrome 79.0.3945.79 on Linux
Script error.
null
@:0:0

but this error occurs a lot when accessing HomeAssistant with the Webos browser.

**Comment by guardianbs** [comment]
HA Core 2024.5.4 and problem persists.

HA continues to be unusable with WebOs browser on LG TV.

**Comment by guardianbs** [comment]
Hooray !!
One year on and the problem appears to be fixed in 2024.8.2
